include/z64.h

```diff
@@ -1178,17 +1178,17 @@ typedef struct PlayState {
     /* 0x11DE4 */ u32 gameplayFrames;
     /* 0x11DE8 */ u8 linkAgeOnLoad;
     /* 0x11DE9 */ u8 unk_11DE9;
-    /* 0x11DEA */ u8 curSpawn;
-    /* 0x11DEB */ u8 numSetupActors;
+    /* 0x11DEA */ u8 spawn;
+    /* 0x11DEB */ u8 numActorEntries;
     /* 0x11DEC */ u8 numRooms;
     /* 0x11DF0 */ RomFile* roomList;
-    /* 0x11DF4 */ ActorEntry* linkActorEntry;
-    /* 0x11DF8 */ ActorEntry* setupActorList;
+    /* 0x11DF4 */ ActorEntry* playerEntry;
+    /* 0x11DF8 */ ActorEntry* actorEntryList;
     /* 0x11DFC */ void* unk_11DFC;
-    /* 0x11E00 */ EntranceEntry* setupEntranceList;
-    /* 0x11E04 */ s16* setupExitList;
-    /* 0x11E08 */ Path* setupPathList;
-    /* 0x11E0C */ ElfMessage* cUpElfMsgs;
+    /* 0x11E00 */ Spawn* spawnList;
```

Comment:
Mostly just a thought but there might be some confusion because of this change since the "spawn" fields/variables are just indexes and not pointers to a `Spawn` struct or an element of the "spawn list". So maybe we should always use `spawnIndex` instead of just `spawn` (just like we use `pathIndex` to avoid confusions with an actual `Path`), or perhaps this list should be named "spawn info list" or something like that.
The types might be enough to avoid this confusion, not sure, but it's something to keep in mind

Reply:
I think the types are enough to rule out confusion

Also spawns are used very little in the code (the `Spawn` type doesn't come up in src/, `spawnList` is referenced only 4 times)
and only in the core engine (not in actors, they really don't need to),
so I think we don't need to worry as much about it as with paths

Also note the comment on spawn vs spawnIndex would only change `PlayState.spawn`, that value is never a temp

All in all I think it's fine as the PR is, but I'd be fine with `PlayState.spawnIndex`, `PlayState.spawnEntryList` or whatever you think may help